articles/openshift/howto-infrastructure-nodes.md

```diff
@@ -358,4 +358,31 @@ Use this procedure for any additional ingress controllers you may have in the cl
     grafana-599d4b948c-btlp2                       3/3     Running   0          2m48s   10.131.4.10   cz-cluster-hsmtw-infra-aro-machinesets-eastus-1-vr56r   <none>           <none>
     kube-state-metrics-574c5bfdd7-f7fjk            3/3     Running   0          2m49s   10.131.4.8    cz-cluster-hsmtw-infra-aro-machinesets-eastus-1-vr56r   <none>           <none>
     ```
-    
+
+### DNS
+
+1. ALlow the DNS pods to run on the infrastructure nodes.
+
+```
+oc edit dns.operator/default
+```
+
+```
+apiVersion: operator.openshift.io/v1
+kind: DNS
+metadata:
+name: default
+spec:
+nodePlacement:
+  tolerations:
+  - operator: Exists
+```    
+
+2. Verify that DNS pods are scheduled onto all infra nodes.
```

Comment:
Please change the number of this step from 2 to 1. Markdown will automatically number the steps correctly when publishing.